src/main/java/com/fruitshop/servicesimpl/ResultMessageImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.fruitshop.servicesimpl;
+
+import com.fruitshop.dao.DataBaseManagerImpl;
+import com.fruitshop.model.Fruit;
+import com.fruitshop.services.ResultMessage;
+import java.util.Map;
+
+public class ResultMessageImpl implements ResultMessage {
+    public static final String COMMA = ",";
+    public static final String TITLE_TO_FILE = "fruit,quantity";
+    private final DataBaseManagerImpl dataBaseManagerImpl = new DataBaseManagerImpl();
+    private StringBuilder sb = new StringBuilder(TITLE_TO_FILE + System.lineSeparator());
```

Comment:
StringBuilder must not be a class field. This object is not a stateless service. Being a field, this object makes your class stateful and can cause some errors.